Runner.php

```diff
@@ -15,34 +16,28 @@ public function __construct($config, $server)
 
     public function queueDirectory($dir, $prefix = '')
     {
-        if(isset($this->config['include_paths'])) {
-            $this->queueWithIncludePaths();
-        } else {
-            $this->queuePaths($dir, $prefix, $this->config['exclude_paths']);
```

Comment:
Agreed, it's somewhat of a delicate situation.

Feel free to ping me when you're ready to chat about it.